readthedocs/templates/base.html

```diff
@@ -296,9 +296,7 @@ <h4>{% trans 'Company' %}</h4>
     </div>
     <!-- END footer-->
 
-    <script>
-      {% block footerjs %}{% endblock %}
-    </script>
+    {% block footerjs %}{% endblock %}
```

Comment:
don't know if makes sense still calling this footerjs anymore p:

Reply:
That's still what it's for, just has the script tag in the block :)